docs/docs/features/encoders.md

```diff
@@ -23,25 +23,17 @@ Keyboards and macropads with encoder support will typically take the two EC11 pi
 
 ### Rotation
 
-Rotation is handled separately as a type of sensor. The behavior for this is set in `sensor-bindings`, which is defined in each keymap layer in the following format:
+Rotation is handled separately as a type of sensor. The behavior for this is set in `sensor-bindings`. See [Sensor Rotation](../behaviors/sensor-rotate.md) for customizing this behavior.
 
-```
-sensor-bindings = <BINDING CW_KEY CCW_KEY>;
-```
-
-- `BINDING`, for now, has only one behavior available; `&inc_dec_kp` for key presses (see [Key Press](../behaviors/key-press.md) for details on available keycodes).
-- `CW_KEY` is the keycode activated by a clockwise turn.
-- `CCW_KEY` is the keycode activated by a counter-clockwise turn.
-
-Additional encoders can be configured by adding more `BINDING CW_KEY CCW_KEY` sets immediately after the first.
+Additional encoders can be configured by adding more bindings immediately after the first.
 
 As an example, a complete `sensor-bindings` for a Kyria with two encoders could look like:
 
 ```
-sensor-bindings = <&inc_dec_kp C_VOL_UP C_VOL_DN &inc_dec_kp PG_UP PG_DN>;
+sensor-bindings = <&rgb_encoder &arrow_encoder>;
 ```
 
-Here, the left encoder is configured to control volume up and down while the right encoder sends either Page Up or Page Down.
+Here, the left encoder is configured to control rgb brightness while the right encoder sends either Left or Right.
```

Comment:
From the `behaviors/sensor-rotate.md` example, users can find the RGB control behavior that `&rgb_encoder` refers to. However, it's not immediately clear that the `&arrow_encoder` behavior is bound to the left/right arrow keys on rotation.
On that note, if the updated sensor-rotate behavior does deprecate the sensor-rotate-kp behavior, would it be more appropriate to combine the main page, `encoders.md`, with the new behavior page you've created in this PR? If so, I'd think a code block with the `&rgb_encoder` and `&arrow_encoder` sensor-rotate behaviors would fit really well here.

Reply:
Another thing I just thought of: if the final decision is to keep the separate `docs/behaviors/sensor-rotate.md`, the behaviors section of `docs/sidebars.js` should also be updated to include the new page, enhancing accessibility.

Reply:
I've added the other encoder into the example code for now.